app/src/main/res/layout/activity_social.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".social.SocialActivity">
+
+    <!-- Top bar -->
+    <include layout="@layout/top_bar"/>
+
```

Comment:
If I remember correctly, there should also be a selector for the time and a switch to toggle between all users and friends. Has this been changed or will it be added later in another PR?

Reply:
Yes you're right. My next task is to implement the friends part of the social activity. I though i implement that button then.